apps/export/views.py

```diff
@@ -26,14 +27,23 @@ def get(self, request, file_id=None, version=None):
         pass
 
 
-class ExportViewSet(viewsets.ReadOnlyModelViewSet):
+class ExportViewSet(mixins.DestroyModelMixin,
+                    mixins.ListModelMixin,
+                    mixins.RetrieveModelMixin,
+                    viewsets.GenericViewSet):
```

Comment:
We can now use `ModelViewSet` here directly. Make sure the permission is defined correctly.

Reply:
But it will also expose, `create` and `update` mixins. Maybe this is okay? @thenav56 @k9845

Reply:
Yeah. But we need `update` for `is_archived`.
And for `create` we could disable it for now or move the `ExportTriggerView` to `ExportViewSet::create`.